ci-automation/test_vendor.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Vendor test runner. Used by test_run inside the mantle container. It
+# sources the vendor test script after doing an initial setup.
+#
+#
+# The initial setup consist of creating the vendor working directory
+# for the vendor test script, specifying the variables described below
+# and changing the current working directory to the vendor working
+# directory.
+#
+#
+# The vendor test script is expected to keep all artifacts it produces
+# in its current working directory.
+#
+#
+# The script specifies the following variables for the vendor test
+# script to use:
+#
+# CIA_VERNUM:
+#   Image version. In case of developer builds it comes with a suffix,
+#   so it looks like "3217.0.0+nightly-20220422-0155". For release
+#   builds the version will be without suffix, so it looks like
+#   "3217.0.0". Whether the build is a release or a developer one is
+#   reflected in CIA_BUILD_TYPE variable described below.
+#
+# CIA_ARCH:
+#   Architecture to test. Currently it is either "amd64" or "arm64".
+#
+# CIA_TAPFILE:
+#   Where the TAP reports should be written. Usually just passed to
+#   kola throught the --tapfile parameter.
+#
+# CIA_CHANNEL:
+#   A channel. Either "alpha", "beta", "stable" or "lts". Usually
+#   passed to kola through the --channel parameter.
```

Comment:
```suggestion
#   A channel. Either "alpha", "beta", "stable" or "lts". Used to
#   find the last release for the update check.
```
Normally there is no channel filtering in kola because we use the filtering by version.